addons/sale_crm/models/crm_lead.py

```diff
@@ -10,23 +10,27 @@ class CrmLead(models.Model):
     _inherit = 'crm.lead'
 
     sale_amount_total = fields.Monetary(compute='_compute_sale_amount_total', string="Sum of Orders", help="Untaxed Total of Confirmed Orders", currency_field='company_currency')
-    sale_number = fields.Integer(compute='_compute_sale_amount_total', string="Number of Quotations")
+    quotation_count = fields.Integer(compute='_compute_sale_amount_total', string="Number of Quotations")
```

Comment:
Maybe we should consider renaming the compute method _compute_sale_information or _compute_sale_data ?